### ISSUE TITLE
Egeria installation fails

### ISSUE THREAD

**Issue by revit13** [IBM/the-mesh-for-data#162]
Running ./hack/install.sh on OpenShift CodeReadyContainer v4.5.14 on Windows 10 with WSL failed due to timeout:
helm install lab ./egeria/open-metadata-resources/open-metadata-deployment/charts/odpi-egeria-lab -f lab.yaml --namespace=egeria-catalog --wait --timeout=8m

Increasing timeout to 30m did not help.

**Comment by roee88** [comment]
Please provide more details, do the pods in egeria-catalog namespace eventually change to Running state? If not, what pod is stuck and in what status?

**Comment by revit13** [comment]
Sorry for my later response on this, I have to turn off my VPN when running CRC ... I forgot to mention that in the 30 min timeout I had also Istio in my cluster as well. This seems to be a resource problem. I did not encounter this problem when running on a remote machine. However on my laptop in Kind cluster on WSL or CRC (Windows 10) I hit the egria timeout. Without Istio ./hack/install.sh usually succeed after second time running it due to the egeria timeout in the first time. When installing egeria manually on fresh cluster I also did not had any problems.
